starfyre/build_main_file_content.py

```diff
@@ -59,7 +59,9 @@ def generate_pages(generated_routes, path):
             result = str(component)
         except ModuleNotFoundError:
             print(f"Error: Could not import module '{{module_name}}'.")
-            continue
+            # throw an error stating that we could not resolve an import
+            raise ImportError('Please resolve your import statements')
```

Comment:
This will throw an error if you put an invalid import statement

Reply:
nicee.